utils/zoo.py

```diff
@@ -23,13 +24,13 @@ def wrapper(stub: str, *args, **kwargs):
 
 def _get_model_framework_file(model, path: str):
     available_files = model.training.default.files
-    transfer_request = 'recipe_type=transfer' in path
+    transfer_request = search("recipe(.*)transfer", path)
```

Comment:
Same as [this comment](https://github.com/neuralmagic/yolov5/pull/86#discussion_r938096000)